server/app/services/export/JsonExporter.java

```diff
@@ -127,6 +136,20 @@ private CfJsonDocumentContext buildJsonApplication(
               jsonApplication.putNull(path);
             }
 
+            break;
+          }
+        case DATE:
```

Comment:
not blocking, for my own information: Does this mean we didn't export dates before?

Reply:
No, just that they were exported using the default logic which uses the same formatting as the application review page. The cases in this switch other than the default override that logic to do formatting specific to the JSON export.